src/components/HighTable/HighTable.module.css

```diff
@@ -1,2 +1,343 @@
+    max-width: 100%;
+    overflow-x: auto;
+  }
+
+  /* cells */
+  th,
+  td {
+    overflow: hidden;
+  }
+
+  /* header */
+  thead {
+    th {
+      background-color: #fff;
+      position: sticky;
+      user-select: none;
+      z-index: 10;
+      top: 0;
+    }
+  }
+
+  /* sortable */
+  thead th[aria-disabled="false"] {
+    cursor: pointer;
+  }
+  th[aria-sort="ascending"]::after,
+  th[aria-sort="descending"]::after {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  th[aria-sort="ascending"]::after {
+    content: "▾";
+  }
+  th[aria-sort="descending"]::after {
+    content: "▴";
+  }
+
+  /* column resize */
+  thead [role="separator"] {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    width: 12px;
+    cursor: col-resize;
+    background-color: #888;
+  }
+
+  /* row numbers */
+  tbody [role="rowheader"] {
+    position: sticky;
+    left: 0;
+    user-select: none;
+    min-width: 48px;
+    max-width: none;
+    width: 48px;
+    cursor: pointer;
+  }
+  tbody [role="rowheader"] span {
+    display: inline;
+  }
+  tbody [role="rowheader"] input {
+    display: none;
+  }
+  /* for selected rows, replace row numbers with checkboxes */
+  tr[aria-selected="true"] [role="rowheader"] span {
+    display: none;
+  }
+  tr[aria-selected="true"] [role="rowheader"] input {
+    display: inline;
+  }
+  /* if selectable, show checkboxes on hover (and focus) */
+  table[aria-multiselectable="true"] tbody [role="rowheader"]:hover span,
+  table[aria-multiselectable="true"] tbody [role="rowheader"]:focus span {
+    display: none;
+  }
+  table[aria-multiselectable="true"] tbody [role="rowheader"]:hover input,
+  table[aria-multiselectable="true"] tbody [role="rowheader"]:focus input {
+    display: inline;
+    cursor: pointer;
+  }
+
+  /* table corner */
+  /* TODO: find a better selector for the table corner */
+  thead td:first-child {
+    position: sticky;
+    cursor: default;
+    min-width: 48px;
+    max-width: none;
+    width: 48px;
+    z-index: 15;
+    top: 0;
+    left: 0;
+  }
+  /* replace corner with checkbox if selection is enabled (read-only or not) */
+  thead td:first-child input {
+    display: none;
+    margin: 0 auto;
+  }
+  thead td:first-child[aria-disabled="false"] {
+    background: #fff; /* redundant with td:first-child, but allows to force the background if it has been overridden with a logo */
+  }
+  thead td:first-child[aria-disabled="false"] span {
+    display: none;
+  }
+  thead td:first-child[aria-disabled="false"] input {
+    display: block;
+  }
+  /* if selectable, show pointer cursor on checkbox */
+  table[aria-multiselectable="true"] thead td:first-child,
+  table[aria-multiselectable="true"] thead td:first-child input {
+    cursor: pointer;
+  }
+
+  /* mock row numbers */
+  .mock-row-label {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    background: var(--mock-row-label-background, #fff);
```

Comment:
nice job getting these to match.

question: why is the fallback #fff instead of transparent or inherit?

Reply:
indeed.

Reply:
The issue with inherit or transparent, for the vertical mock row label, or for the header row, is that we see the cells scrolling below them, while they keep in place due to "position: sticky". In the theme, they have an opaque background that ensures they are on top, and the cells are hidden when sliding behind.

So... #fff was a way to show, even in unstyled mode, that the vertical and horizontal headers should be on top.

But let's see, I'm changing them to transparent (as inherit is a bit too uncertain to me: it could contain anything, and it's not necessarily related).